provider/pkg/gen/overlays.go

```diff
@@ -1182,7 +1228,7 @@ var yamlConfigGroupV2Resource = pschema.ResourceSpec{
 					},
 				},
 			},
-			Description: "Set of paths or a URLs that uniquely identify files.",
+			Description: "Set of paths or a URLs that uniquely identify files. Accepts glob patterns.",
```

Comment:
Let's move this to #2844 since we already have a PR open for ConfigGroup changes.